package.json

```diff
@@ -7,7 +7,10 @@
   ],
   "type": "module",
   "main": "index.cjs",
-  "module": "index.js",
```

Comment:
Wouldn't removing this break some existing tooling. I'm thinking about things like rollup, webpack, etc.. tools that walk the node_modules and choose an entry-point based on this field. Those tools may not have updated to use this new nodejs-specific thing.

Reply:
It could. I am reverting it.